config/initializers/airbrake.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# Airbrake is an online tool that provides robust exception tracking in your
+# Rails applications. In doing so, it allows you to easily review errors, tie an
+# error to an individual piece of code, and trace the cause back to recent
+# changes. Airbrake enables for easy categorization, searching, and
+# prioritization of exceptions so that when errors occur, your team can quickly
+# determine the root cause.
+#
+# Configuration details:
+# https://github.com/airbrake/airbrake-ruby#configuration
+if (project_id = ENV['AIRBRAKE_PROJECT_ID']) &&
```

Comment:
We don't usually put an `airbrake.rb` config in our project. Was there something in here that you had to update from the default?

Reply:
After I created the blazer errbit in the directions it said to run dip rails g airbrake and i think it came from that command. Should I remove this file?

Reply:
Yes, go ahead and remove it. It has some duplicate settings from our `errbit.rb`